tool/resources/org/antlr/v4/tool/templates/codegen/Python3/Python3.stg

```diff
@@ -829,7 +874,7 @@ class <lexer.name>(<if(superClass)><superClass><else>Lexer<endif>):
 SerializedATN(model) ::= <<
 def serializedATN():
     return [
-        <model.serialized: {s | <s>}; separator=",", wrap>
+        <model.serialized: {s | <s>}; separator=", ", wrap>
```

Comment:
I found this rule to be close to what I wanted, except that there remains a trailing whitespace when a line is wrapped. I didn't know how to fix this, and figured it was good enough for now.